products/views.py

```diff
@@ -77,26 +77,26 @@ def get(self,request):
 class ProductlistView(View):
     def get(self,request,sub_category_id):
         try:
+            ALL_PRODUCTS = 0
             products = Product.objects.filter(sub_category__id=sub_category_id)
+            
+            if sub_category_id == ALL_PRODUCTS:
+                products = Product.objects.all()
 
             if not products:
```

Comment:
```suggestion
            if not products.exists():
```
위와 같이 exists() 를 추가하여, 존재여부를 바로 true, false 한 결과로 확인하는 것을 추천합니다! 
빈 쿼리셋이 falsy 한 값이긴 하지만 보다 직관적으로 exists 를 추가해주시는 것이 좋습니다!

Reply:
수정했습니다!